src/GSvar/MainWindow.cpp

```diff
@@ -3671,6 +3671,8 @@ void MainWindow::loadFile(QString filename, bool show_only_error_issues)
 				// check for required files
 				foreach (const QString& rna_ps_id, db.getValues("SELECT id FROM processed_sample WHERE sample_id=:0", QString::number(rna_sample_id)))
 				{
+					if (!db.userCanAccess(LoginManager::userId(), rna_ps_id.toInt())) continue;
```

Comment:
The menu is enabled above. It should be disabled if there is no accessable RNA.

Reply:
I was not sure about the menu. Do you mean that all the items on the menu should be disabled? Even the one the screenshot?
![image](https://user-images.githubusercontent.com/5293569/199940728-3ccc13be-9642-4c74-b051-33b5c71f1148.png)

Reply:
No, this entry should be active.

But then the line "rna_menu_btn_->setEnabled(true);" can be removed.
And all other entries should be disabled by default when GSvar starts.

Reply:
I have adjusted the menu code